projects/jdwp/codegen/new_type_generator.py

```diff
@@ -0,0 +1,21 @@
+"""New type Generator."""
+from projects.jdwp.defs.schema import PrimitiveType
+
+
+def get_python_type(jdwp_type: str) -> str:
+    """Map JDWP type to Python type."""
+    mapping = {
+        "string": "str",
+        "int": "int",
+        "byte": "bytes",
```

Comment:
int and byte are now moved away to `IntegralType`, I guess it's fine to use raw `int` for them rather than defining a new type.